src/server/wtr-config.js

```diff
@@ -196,6 +196,7 @@ export class WTRConfig {
 		config.groups.forEach(g => g.browsers = this.getBrowsers(g.browsers));
 
 		if (open || watch) {
+			config.testsFinishTimeout = 15 * 60 * 1000;
```

Comment:
This is a bit unrelated, but when I was trying to debug my new tests with `--open`, I only had two minutes before the whole thing force-closed.  This ups that timeout to 15 minutes in `open` or `watch` mode.